engines/stark/ui/menu/saveloadmenu.cpp

```diff
@@ -113,7 +131,16 @@ void SaveMenuScreen::open() {
 }
 
 void SaveMenuScreen::onSlotSelected(int slot) {
-	g_engine->saveGameState(slot, "TestSave");
+	int chapter = StarkGlobal->getCurrentChapter();
+
```

Comment:
What does the returned value of `getCurrentChapter()` really mean? I thought it was the index of the chapter, but calling this function in *Chapter 1* gives me value `10`.